### PR TITLE
Starter Kit for SurrealDB + SpringBoot By PDROJACK

### PR DESCRIPTION
**Pull request by PDROJACK**
- Starter Kit for SurrealDB + SpringBoot By PDROJACK

### CODE REVIEW

**Comment by aravindputrevu** [comment]
@PDROJACK I took a quick look at the code. Here are my quick suggestions :- 

* Please add a clearer readme listing all the endpoints and the way to call them - todo, todo/{id}  
* It looks like the result is shown with binary blob, can you please return the item added? A screenshot in the readme would be good too! 
* Finally, some optional comments - would appreciate if you could use SurrealQL anywhere, say using something like record ids.

File: README.md [hunk 1]
```diff
@@ -141,6 +141,7 @@ If you want to contribute to this list, then please read the [contributing guide
 - [SurrealDB + SolidStart Starter](https://github.com/metruzanca/surreal-solid-template) - Sam "metru" Zanca.
 - [SurrealDB + SvelteKit Starter](https://github.com/spinspire/surrealdb-sveltekit-starter) - Jitesh Doshi.
 - [Starter Kit for SurrealDB + Tauri + Next.js](https://github.com/reymom/surrealdb-starter-taurikit) - Reymom.
+- [Starter Kit for SurrealDB + SpringBoot](https://github.com/PDROJACK/surrealdb-springboot-starter) - PDROJACK.
```
Comment (aravindputrevu) [inline README.md:144]: @PDROJACK made some comments on the issue.

**Comment by PDROJACK** [comment]
> @PDROJACK I took a quick look at the code. Here are my quick suggestions :-
> 
> * Please add a clearer readme listing all the endpoints and the way to call them - todo, todo/{id}
> * It looks like the result is shown with binary blob, can you please return the item added? A screenshot in the readme would be good too!
> * Finally, some optional comments - would appreciate if you could use SurrealQL anywhere, say using something like record ids.

Hey @aravindputrevu  I have made some changes considering your suggestions. I've added open API spec and swagger-UI as well to document the API endpoints. For the last point, I am using SurrealQL while deleting and getting items by id.

**Comment by aravindputrevu** [comment]
@PDROJACK Thanks for the improvements. Kindly add `postman` reference to the `readme.md` as well as a line about - docker compose is pulling `surrealdb`. It'd be a great resource if you add a list of features you are using in SurrealDB. 

Again, appreciate and thanks for the contribution.

**Comment by naisofly** [comment]
Well done @PDROJACK!
Thank you for contributing to SurrealDB during Hacktoberfest 2023 🎉
Kindly send an email to [community@surrealdb.com](mailto:community@surrealdb.com) to coordinate the delivery of your SurrealDB swag! 😁

**Pull request merged**